packs/st2cd/actions/st2_finalize_release.meta.yaml

```diff
@@ -0,0 +1,19 @@
+---
+  name: "st2_finalize_release"
+  runner_type: "action-chain"
+  description: "Look at workflows/st2_finalize_release.yaml"
+  enabled: true
+  entry_point: "workflows/st2_finalize_release.yaml"
+  parameters:
+    st2_repo:
+      type: "string"
+      description: "Url of the st2 repo to clone"
+      default: "https://github.com/StackStorm/st2.git"
+    st2_repo_target:
+      type: "string"
+      default: "/home/stanley/st2"
+      description: "Path to clone st2 repo to."
+    branch:
+      type: "string"
+      description: "st2 branch to tag."
```

Comment:
(sorry for the nit pick)

Usually it's better to explicitly specify a revision when creating a tag. In this case it looks like we are just checking out this specific branch and tagging a latest revision in it.

I don't want to bother you further since using an exact revision might be painful (getting it all the way to the workflow), but right now there is a potential race so just something to keep in mind.